libs/execution_context/ebpf_core.c

```diff
@@ -760,59 +886,100 @@ typedef struct _ebpf_protocol_handler
     size_t minimum_reply_size;
 } const ebpf_protocol_handler_t;
 
-static ebpf_protocol_handler_t _ebpf_protocol_handlers[EBPF_OPERATION_GET_PROGRAM_INFORMATION + 1] = {
```

Comment:
Can we leave this as max enum + 1? This ensures that the number of entries in the array is equal to the number of enum values.

Reply:
Defining a max enum is bad programming practice and windows guidelines say to not do that.

Reply:
So, what is the best option for making sure the array and enums remain in synch?

Reply:
Over time, enum values might be deprecated and so specific values might be treated the same as illegal values.  As such, you have to allow for this at runtime anyway.  So simply testing an input for <0 or >_countof(array) for example can be done at runtime.

Reply:
I did not want  EBPF_OPERATION_GET_MAP_INFORMATION+1 as that would mean changing the array size every time a new enum value were added. I introduced a  EBPF_OPERATION_MAX but backed that out on Dave's comment. So currently leaving this as []. I am not quite sure what the consensus solution is.

Reply:
[] is current best practice